plugins/BEdita/Core/src/Model/Action/SignupUserAction.php

```diff
@@ -357,11 +358,11 @@ protected function checkExternalAuth(array $data)
      * @return array Response from an OAuth2 provider
      * @codeCoverageIgnore
```

Comment:
Why is this ignored? If we're unable or not willing to cover a method it's better just to leave it uncovered, rather than cheating… 😉

Reply:
Yeah, we should definitely test it... but I agree we should remove the ignore tag until we've done it.